modules/apps/accessibility/accessibility-menu-web/src/main/java/com/liferay/accessibility/menu/web/internal/util/AccessibilitySettingsUtil.java

```diff
@@ -47,6 +54,33 @@ public static List<AccessibilitySetting> getAccessibilitySettings(
 						ACCESSIBILITY_SETTING_SHOW_UNDERLINE)));
 	}
 
+	public static boolean isAccessibilityMenuEnabled(
+		HttpServletRequest httpServletRequest,
+		ConfigurationProvider configurationProvider) {
+
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		try {
+			AccessibilityMenuConfiguration accessibilityMenuConfiguration =
+				configurationProvider.getGroupConfiguration(
+					AccessibilityMenuConfiguration.class,
+					themeDisplay.getScopeGroupId());
+
+			if (accessibilityMenuConfiguration.enableAccessibilityMenu()) {
+				return true;
+			}
```

Comment:
Hey @ethib137, you can just return the `accessibilityMenuConfiguration.enableAccessibilityMenu()` result here instead of using an if statement. Thx